exonum-java-binding/core/rust/src/runtime/java_service_runtime.rs

```diff
@@ -252,6 +251,44 @@ impl JavaServiceRuntime {
     }
 }
 
+/// Wrapper for InitArgsBuilder.
+struct JvmArgsBuilder {
+    internal: InitArgsBuilder,
+    jvm_args_line: String,
+}
+
+impl JvmArgsBuilder {
```

Comment:
Why does this code manually accumulates the options that are available in the builder (and creates a separate structure for that) instead of:
- Adding InitArgsBuilder#toString
- Adding InitArgsBuilder#getOptions?

Probably, InitArgs#toString would be even better, but that structure is rather low-level.

Reply:
Well, I was also thinking about both of these options but:
- InitArgsBuilder#build could fail and toString/Display/Debug would print faulty data in this case.
- InitArgs#toString looks handy but it is really low-level.
- Also, I was hoping to refactor the service runtime code by moving this set of associative functions for arguments processing to new arguments builder which looks more natural but was unable to create more clear code at first attempt.

Reply:
> InitArgsBuilder#build could fail and toString/Display/Debug would print faulty data in this case.

Not sure I understand what you mean by that — how could they print faulty data? Also, it seems that the only reason for it to fail is if the passed strings contain 0 byte:

> [from CString::new, used in `#build`] This function will return an error if the supplied bytes contain an internal 0 byte. The NulError returned will contain the bytes as well as the position of the nul byte.

---

From the maintanence perspective, the added builder seems like an overkill, at least in its present form — instead of two simple functions (that might be considered too simple to break) with a clear purpose, we have a structure with unclear purpose, with functions doing several things at once (e.g., building a string repr of the arg line and appending these arguments to the InitArgsBuilder), and no tests. 

If not `InitArgsBuilder#Display/Debug`, would accumulating arguments in a list first, then logging, then adding to the builder be simpler?